app/src/main/res/layout/fragment_sessions.xml

```diff
@@ -85,6 +86,21 @@
                 android:textColor="@color/white"
                 />
 
+        <FrameLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="@color/white"
```

Comment:
If the background is undefined, the date text view with empty text and gray background will be shown. Is it ok?
![device-2017-02-06-172056](https://cloud.githubusercontent.com/assets/900756/22639298/a77d2cc6-ec90-11e6-9102-f016a47ce322.png)